recipe/conda_build_config.yaml

```diff
@@ -536,7 +536,7 @@ openjpeg:
 openmpi:
   - 4.0
 openssl:
-  - 1.1.1a
+  - 1.1.1d
```

Comment:
```suggestion
  - 1.1.1
```

Reply:
Do we not need rebuilds for the alphabet changes?

Reply:
No rebuilds are needed, openssl 1.1.1 is (or at least has been) ABI compatible between releases.  The package has a [run_export](https://github.com/conda-forge/openssl-feedstock/blob/master/recipe/meta.yaml#L40-L43) which adds the appropriate constrain to downstream packages which have openssl in the host section.

Reply:
@isuruf's suggested change is a good one, it will result in builds using the latest openssl package from the 1.1.1 line.  Without this the pinning will need to be updated after each release, e.g. 1.1.1e, 1.1.1f, ....